### ISSUE TITLE
"Change image checksum" doesn't work 

### ISSUE THREAD

**Issue by a84r7a3rga76fg** [K1rakishou/Kuroba-Experimental#1078]

**Comment by bqv** [comment]
It does, what are you doing specifcally.

**Comment by a84r7a3rga76fg** [comment]
No, it does not. It used to generate a new image hash that was rarely used before. Now it generates the same hash.

If you will keep insisting it works fine, then record yourself doing this:
Make a post.
Attach an image.
Click on "change image checksum" once.
Submit the post.
Repeat the 4 steps again.

The two pictures are going to have the same hash. It didn't do this before.
